PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,201 @@
+            Get
+
+            The get verb will retrieve an instance, or instances, of the specified noun.
+
+            So for example:
+
+                Get-Service
+
+            Will retrieve all the services running on your current machine.
+        #>
+
+        It 'is for commands that retrieve data'  {
+            <#
+                Using the Get-Command cmdlet, which retrieves all the cmdlets currently available,
+                find 5 new get commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Get cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Get).Name
+        }
+    }
+
+    Context "New" {
+         <#
+            New
+
+            The New verb will create an instance of the given noun.
+
+            So for example 'New-GUID' will create a new GUID, or 'New-LocalUser' will create a local
+            user on your machine.
+
+            Before continuing with this excerised, you're advised to run the following code:
+
+                $Path = "YOUR PATH.txt"
+                New-Item -Path $ItemPath -ItemType file
+
+            This will create a new text file, in the location your specify, and we can use this
+            to explain the other verbs later on.
+        #>
+
+        It 'is for commands that create data' {
+            <#
+                Using the Get-Command cmdlet, which retrieves all the cmdlets currently available,
+                find 5 new new commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a New cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb New).Name
+
+        }
+
+    }
+
+    Context "Add" {
+        <#
+            Add
+
+            The Add verb will append information to a given noun.        
+
+            If you followed the example in the 'new' test, you can use add to add some
+            text to your newly created text file:
+            
+                $Path = "YOUR PATH.txt"
+                Add-Content -Path $Path -Value "Sgt. Bash is the best house robot because... fire."
+            
+            Before continuing, run this command several times. See what happens, is it the
+            result you expected?
+
+            You'll see that several lines of text were added to the file. This cmdlet only appends
+            information, it does not overwrite. 
+
+            Basically, if it doesn't exist then a cmdlet with the add verb can probably be used
+            to make it so.
+
+            A classic example that I've seen a lot in my time is office 365 calendar permissions.
+            If you want to grant permissions to somebody who doesn't have any, you use Add. If 
+            you want to change permissions for somebody who already has some, you need to use a
+            different verb that we'll cover later.
```

Comment:
Rephrased in next commit. Let me know your thoughts 🦄

Reply:
@vexx32 let me know if the rephrase is OK?

Reply:
Added some comments to sections of it. You could probably just remove this last sentence here without losing anything, since you continue the example further down?